buildSrc/src/main/groovy/org/elasticsearch/gradle/plugin/PluginBuildPlugin.groovy

```diff
@@ -211,35 +211,18 @@ public class PluginBuildPlugin extends BuildPlugin {
 
     /** Adds nebula publishing task to generate a pom file for the plugin. */
     protected static void addClientJarPomGeneration(Project project) {
-        project.plugins.apply(MavenPublishPlugin.class)
-
-        project.publishing {
-            publications {
-                clientJar(MavenPublication) {
-                    from project.components.java
-                    artifactId = project.pluginProperties.extension.name + '-client'
-                    pom.withXml { XmlProvider xml ->
-                        Node root = xml.asNode()
-                        root.appendNode('name', project.pluginProperties.extension.name)
-                        root.appendNode('description', project.pluginProperties.extension.description)
-                        root.appendNode('url', urlFromOrigin(project.scminfo.origin))
-                        Node scmNode = root.appendNode('scm')
-                        scmNode.appendNode('url', project.scminfo.origin)
```

Comment:
Are you sure this isn't needed anymore? IIRC I added it because the scm info didn't get into the pom correctly at the time, and caused validation to fail when publishing to maven central.

Reply:
I do remember doing diffs of before and after XMLs and didn't notice anything odd. Any edge cases or additional tests I should do ?

Reply:
I just checked this again to make sure:
```
[:~/work/elastic/elasticsearch/modules/reindex] fix-multi-jar-publishing-take-2+ ± diff ./build/distributions/reindex-client-7.0.0-alpha1-SNAPSHOT.pom ~/tmp/reindex-client-7.0.0-alpha1-SNAPSHOT.pom.master
2c2
< <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
---
> <project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```
So the change produces no effect on the scm info in the pom.

Reply:
Have you run the build in a non git checkout?

Reply:
I have not, but is that relevant for the poms we submit to maven or we just want to make sure it doesn't break ?

Reply:
This does work with non git and just produces: 
```
  <scm>
    <url/>
  </scm>
```
This is consistent with master.